compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java

```diff
@@ -5432,9 +5432,12 @@ private BLangMarkdownDocumentation createMarkdownDocumentationAttachment(Optiona
 
                     BLangIdentifier paraName = new BLangIdentifier();
                     Token parameterName = parameterDocLineNode.parameterName();
-                    paraName.value = parameterName.isMissing() ? "" : parameterName.text();
+                    var parameterNameValue = parameterName.isMissing() ? "" : parameterName.text();
```

Comment:
We also need to replace unicode code points with respective unicode characters.
```suggestion
                    var parameterNameValue = parameterName.isMissing() ? "" : IdentifierUtils.unescapeUnicodeCodepoints(parameterName.text());
```

Try this example.
```bal
# + 'foo_\u{2345}\u{0376} - The error struct to be logged
function foo(error 'foo_\u{2345}\u{0376}) {}
```

Reply:
@KRVPerera shall we add this sample as a test case too. We don't have a test case for `IdentifierUtils.unescapeUnicodeCodepoints()` filter.

Reply:
added a test